src/w2field.js

```diff
@@ -492,7 +494,7 @@ class w2field extends w2base {
                 }, 1)
             }
             // if readonly or disabled
-            if (query(this.el).prop('readonly') || query(this.el).prop('disabled')) {
+            if (query(this.el).prop('readOnly') || query(this.el).prop('disabled')) {
```

Comment:
Strictly, it is not necessary because of this: https://github.com/vitmalina/mquery/blob/main/src/mquery.js#L83. JQuery also does it.

Reply:
Ah I thought it wasn't picking it up, good to know!